src/plugins/intel_gpu/src/graph/graph_optimizer/mark_shape_of_subgraphs.cpp

```diff
@@ -26,6 +27,10 @@ void mark_shape_of_subgraphs::look_for_shape_of_subgraph(program_node& node) {
     bool has_shape_of_subgraph_dep = false;
     for (auto& dependency : node.get_dependencies()) {
         if (dependency.first->is_in_shape_of_subgraph()) {
+            // skip mark_node for broadcast node if dependency node is shape_of
+            if (dependency.first->is_type<shape_of>() && node.is_type<broadcast>()) {
+                break;
```

Comment:
Isn't this too aggressive? Shouldn't we check perf impact?

Reply:
Hi @yeonbok , changed to check whether fake quantization is used for reorder node after convolution node (quantized model case), and skip to mark the reorder node in shape_of flow in commit 3e86c67, to avoid can't get implementations for reorder node (Convolution_0_reorder_1).

Reply:
No `data + shape_of -> broadcast` pattern found in LLM (like chatglm3/qwen-7b-chat/llama-3-8b/gemma-7b), and no performance drop found after applying commit [1fd12ac](https://github.com/openvinotoolkit/openvino/pull/27786/commits/1fd12aceb82c8fc7aada3dabfb139700e3a2931c).

![image](https://github.com/user-attachments/assets/faa64608-6099-4773-8e76-01cc87f48c97)